examples/Patterns/Saga/Program.cs

```diff
@@ -22,11 +24,11 @@ public static void Main(string[] args)
             var busyProbability = 0.01;
             var provider = new InMemoryProvider();
 
-            var props = Actor.FromProducer(() => new Runner(numberOfTransfers, uptime, refusalProbability, busyProbability, retryAttempts, false))
+            var props = Props.FromProducer(() => new Runner(numberOfTransfers, uptime, refusalProbability, busyProbability, retryAttempts, false))
                 .WithChildSupervisorStrategy(new OneForOneStrategy((pid, reason) => SupervisorDirective.Restart, retryAttempts, null));
             
             Console.WriteLine("Spawning runner");
-            var runner = Actor.SpawnNamed(props, "runner");
+            var runner = Context.SpawnNamed(props, "runner");
```

Comment:
I'm not sure if this is correct. please check, thanks.

Reply:
It's a correct replacement. As mentioned above though it's prolly simpler to just use RootContext.Empty.

Reply:
Done. Global `Context` is now set to `RootContext.Empty`.